apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -2010,7 +2010,7 @@ module.exports = {
       // tl;dr: don't cache any results that would create course issues.
       const hasCourseIssues = computedData?.courseIssues?.length > 0;
       if (cacheKey && !hasCourseIssues) {
-        cache.set(cacheKey, computedData);
+        cache.set(cacheKey, computedData, config.questionRenderCacheTtlSec * 1000);
```

Comment:
The tradeoff here: we'd previously cache things indefinitely, assuming no memory pressure. This means that we'll recompute question panels *slightly* more often than before, which I think is ok.